lstchain/calib/camera/calibrator.py

```diff
@@ -114,7 +112,6 @@ def _initialize_correction(self):
 
         try:
             with HDF5TableReader(self.calibration_path) as h5_table:
-                assert h5_table._h5file.isopen == True
```

Comment:
what is the problem on checking here whether the files is opened?

Reply:
that check is already performed by pytable, so it is redondant